hw/ip/flash_ctrl/data/flash_ctrl.hjson

```diff
@@ -302,25 +302,59 @@
   regwidth: "32",
   registers: {
     core: [
-      { name: "FLASH_DISABLE",
+      { name: "DIS",
```

Comment:
I'd prefer to rename the register `DISABLE` unless `DIS` is more conventional.

Reply:
ah the problem is `DISABLE` is a system verilog key word... i actually originally had that.
`FLASH_DISABLE` turns into `FLASH_CTRL_FLASH_DISABLE`, which seems a bit redundant..